doc/src/manual/mathematical-operations.md

```diff
@@ -1,5 +1,5 @@
 # Mathematical Operations and Elementary Functions
-
+ 
```

Comment:
The whitespace here was probably accidental?

Reply:
Definitely